packages/api/lambdas/sf-event-sqs-to-db-records/index.js

```diff
@@ -121,6 +121,35 @@ const writeRecords = async ({
   });
 };
 
+async function formatCumulusDLQMessage(message, error) {
+  const errorString = error.toString();
+  let executionEvent;
+  try {
+    executionEvent = parseSQSMessageBody(message);
+  } catch {
+    executionEvent = null;
+  }
+  const execution = executionEvent?.detail?.executionArn || 'unknown';
+  const stateMachine = executionEvent?.detail?.stateMachineArn || 'unknown';
+
+  let cumulusMessage;
+  try {
+    cumulusMessage = await getCumulusMessageFromExecutionEvent(executionEvent);
+  } catch {
+    cumulusMessage = null;
+  }
+  const collection = cumulusMessage?.meta?.collection?.name || 'unknown';
+  const granules = cumulusMessage?.payload?.granules?.map((granule) => granule?.granuleId || 'unknown') || 'unknown';
+  return {
```

Comment:
So - `message` is currently of type `@cumulus/types/CumulusMessage`.   I *think* it's ok to change it to a different type (and noting this isn't yet typescripted) to add more fields, but there's a user contract here with respect to what *they* are expecting from a Cumulus message. 

Is there a good reason this couldn't/shouldn't be it's own object like 


{ 
   message: <CumulusMessage>
   <other things>:
}

Other than it requires changes to anything parsing the DLQ message?

Reply:
As pointed out in slack, ^ isn't correct.    Based on the typings from the method here, it should be `EventBridgeEvent`. 

The concern here really is that this update is great, but because there are format assumptions inherent in the <thing> that's being put into the queue, we need to be sure that the downstream typings are updated (it's really unfortunate this isn't typed, yet) and that contracts are honored.     Apologies for the misdirection on the CumulusMessage type.

Reply:
as of now, the DLQ message changes have been minimalized to capturing just the triggering error
the write-db-dlq-records-to-s3 lambda now expects (but defaults if not found) that error string and also parses out other data from the message body to put it at the top level.
this should be very similar, but it turned out once I investigated further that write-db-dlq-records-to-s3 was nesting the contents of the sqs message one layer deeper, such that I had to pull everything out a layer anyway.

Reply:
Removing this comment as outdated/work has been done